opensrp-app/src/main/java/org/smartregister/util/HttpResponseUtil.java

```diff
@@ -35,6 +36,9 @@ public static LoginResponseData getResponseBody(HttpResponse response) {
             if (StringUtils.isBlank(responseString)) {
                 return null;
             }
+
+            Utils.saveCampaignOperationalAreas(new JSONObject(responseString));
```

Comment:
why not extract attributes from  LoginResponseData  returned by this method object instead of the JSON. 

its faster to process/traverse a java object than using SimpleJSON methods to traverse the JSON string